image-tools-interface.md

```diff
@@ -0,0 +1,49 @@
+# Image Tools Interface
+
+The below document describes an interface for the OCI Image Tools project as desired by the OCI Certification Program working group (Cert WG.)
+
+Goals:
+* An engineer can verify a container image complies with the OCI Image Spec (https://github.com/opencontainers/image-spec/releases) 
+
+Inputs:
+* (REQUIRED): Path or reference to a container image
+* (OPTIONAL): If the user would like verbose output
+
+Outputs:
+* Number of cases run
+* Number of cases passed
+* Number of cases failed
+* List of cases failed
+* Version of OCI Image Tools
```

Comment:
> and maybe git commit of the tool?

*I* like this, but note that in a similar context the OCI consensus (as voiced by @duglin [here][1]) was that a consistent API for extracting the implementation version was not required for interop.

> And the timestamp the validation just ran.

Does this matter?  Hopefully the version of the test suite/tools and the digest(s) of the tested object would be sufficient without recourse to the validation timestamp.  Higher-level tooling can always add a timestamp if it feels it would be useful.

> Also, presumably this would need to output the digests of the objects tested.

This is tricky for stuff outside of CAS.  For example, if validation was pointed at an image-layout via a HTTP URL, or at a directory, what digest hash do you show for complaining about “I couldn't find `index.json` there”?

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2016/opencontainers.2016-07-20-21.03.log.html#l-75